public/app/core/components/Select/DashboardPicker.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { PureComponent } from 'react';
 import { AsyncSelect } from '@grafana/ui';
+import { SelectableValue } from '@grafana/data';
 import { debounce } from 'lodash';
 import { getBackendSrv } from 'app/core/services/backend_srv';
 import { DashboardSearchHit, DashboardDTO } from 'app/types';
 
 export interface Props {
   className?: string;
   onSelected: (dashboard: DashboardDTO) => void;
+  current: SelectableValue<string>;
```

Comment:
think for other form/picker components we use value prop name. Should this be nullable for the default/empy state?  

Why is the value string?

Reply:
I looked at DataSourcePicker which has `current`, I don't have a strong opinion on name.

seems to work if I provide undefined

changed value to number, think I was playing around with using dashboardUid and forgot to change.